java/src/org/openqa/selenium/grid/log/TerseFormatter.java

```diff
@@ -95,7 +95,7 @@ public synchronized String format(final LogRecord record) {
     return buffer.toString();
   }
 
-  private String levelNumberToCommonsLevelName(Level level) {
+  private static String levelNumberToCommonsLevelName(Level level) {
```

Comment:
Why is this changed?

Reply:
Oh, that wasn't intentional. Sorry, I'll remove and resubmit.

Reply:
Thank you